frontend/micro-ui/web/micro-ui-internals/packages/libraries/src/hooks/useCustomAPIHook.js

```diff
@@ -2,8 +2,9 @@ import { useQuery, useQueryClient } from "react-query";
 import { CustomService } from "../services/elements/CustomService";
 
 
-const useCustomAPIHook = (url, params, body, config = {}) => {
+const useCustomAPIHook = ({url, params, body, config = {},jsonPath}) => {
   const client = useQueryClient();
+  
   const { isLoading, data } = useQuery(
     ["CUSTOM", { ...params,...body.Projects }].filter((e) => e),
```

Comment:
@nipunarora-eGov  since we already use loadash functions 
try this part with this 
https://www.geeksforgeeks.org/lodash-_-clonedeep-method/
so don't add json path

Reply:
Okay @jagankumar-egov will try this